src/List/columns/selectors.js

```diff
@@ -0,0 +1,11 @@
+export const getColumnsForModelType = (state, modelType) => {
+    const m = state.configurableColumns.modelTypes[modelType];
+    if (m && m.columns) {
+        return Array.isArray(m.columns) ? m.columns : [];
+    }
+    return [];
+};
```

Comment:
I agree. The reason this is here, is because while developing, I managed to save what should be an array to the server, and this was sort of a quick fix for that - and I forgot to change it. I think this sort of logic belongs in the action creators.

Reply:
👍